<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1746</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Background principal */
QWidget {
    background-color: #e5e7e6;
    color: #e5e7e6;
}

/* Boutons */
QPushButton {
    selection-color: red;
    border-radius: 10px;
    padding: 10px;
    color: white;
    font: bold 14px &quot;Segoe UI&quot;;
}

QPushButton:hover {
    background-color: #DAAB3A;  
    border-color: #607890;
 
}

QPushButton:pressed {
    background-color: #DAAB3A; 
}

/* Labels */
QLabel {
    font: bold 16px &quot;Segoe UI&quot;;
    color: #ffffff;
    background-color: transparent;
    text-align: center;
}

/* Frames */
QFrame {
    background-color: #9ab4ce;
    border: 2px solid #607890;
    border-radius: 10px;
}

/* Progress Bars */
QProgressBar {
    border: 2px solid #607890;
    border-radius: 5px;
    background: #2e2e3e;
    text-align: center;
    color: white;
    font: bold 12px &quot;Segoe UI&quot;;
}

QProgressBar::chunk {
    width: 20px;
}

/* Text Browsers */
QTextBrowser {
    border: 2px solid #607890;
    border-radius: 10px;
    background-color: #9ab4ce;
    color: white;
    font: 12px &quot;Segoe UI&quot;;
    padding: 10px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="headerFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="stratButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/img/jouer.png</normaloff>:/img/jouer.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/img/pause.png</normaloff>:/img/pause.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/img/bouton-darret.png</normaloff>:/img/bouton-darret.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="accelButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/img/avance-rapide.png</normaloff>:/img/avance-rapide.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="timeButton">
         <property name="text">
          <string>Heure : 00 00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="alertButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/img/cercle.png</normaloff>:/img/cercle.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="livraisonButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/img/inventaire (1).png</normaloff>:/img/inventaire (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="paramButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/img/parametres.png</normaloff>:/img/parametres.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="leftActionButtonsFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTextBrowser" name="stockInfoTextBrowser">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>90</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:12px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Niveaux de stock:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;- Ingrédient 1: 50&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;- Ingrédient 2: 30&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;- Ingrédient 3: 20&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Commandes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="orderList">
            <item>
             <property name="text">
              <string>Nouvel élément</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nouvel élément</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nouvel élément</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nouvel élément</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nouvel élément</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="diningAreaFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="diningAreaLabel">
            <property name="text">
             <string>Salle de restauration</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGraphicsView" name="diningAreaView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>600</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>600</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QGraphicsView {
    background-image: url(:img/solee.jpg);
    background-repeat: no-repeat;
    background-position: center;
}</string>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
            </property>
            <property name="backgroundBrush">
             <brush brushstyle="NoBrush">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>diningAreaView</zorder>
         <zorder>diningAreaLabel</zorder>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="kitchenAreaFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="kitchenAreaLabel">
            <property name="text">
             <string>Cuisine</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGraphicsView" name="kitchenAreaView">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>600</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QGraphicsView {
    background-image: url(:img/solee.jpg);
    background-repeat: no-repeat;
    background-position: center;
}</string>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>kitchenAreaView</zorder>
         <zorder>kitchenAreaLabel</zorder>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="rightInfoFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="customerMoodButton">
            <property name="text">
             <string>Menu du restaurant</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="menuListWidget">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Salade</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Glace avec Creme</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jus de raison</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="tableInfoTextBrowser">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>100</height>
             </size>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:12px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Infos tables:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;- Total tables: 20&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;- Tables occupées: 10&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;- Tables disponibles: 10&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="customerInfoTextBrowser">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>100</height>
             </size>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:12px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Infos clients:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;- Total clients: 50&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;- Clients en attente: 5&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;- Clients servis: 45&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="dashboardFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="stockLevelsLabel">
         <property name="text">
          <string>Niveaux de stock</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="stockLevelsProgressBar">
         <property name="value">
          <number>70</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tableSeatsLabel">
         <property name="text">
          <string>Places à table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="tableSeatsProgressBar">
         <property name="value">
          <number>80</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="clientMoodLabel">
         <property name="text">
          <string>🙂</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="clientMoodProgressBar">
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
